config/examples/Formbot/T_Rex_2+/Configuration.h

```diff
@@ -1661,7 +1661,7 @@
 // @section extruder
 
 // For direct drive extruder v9 set to true, for geared extruder set to false.
-#define INVERT_E0_DIR true
+#define INVERT_E0_DIR false
```

Comment:
See https://github.com/MarlinFirmware/Configurations/issues/865#issuecomment-1352042039. This machine shipped with a few different hardware revisions, so this will break the config for someone else.

This config really needs some flags at the top to change the drivers & invert axis directions based on driver types.